src/org/opendatakit/briefcase/export/ExportConfiguration.java

```diff
@@ -149,12 +161,25 @@ public ExportConfiguration setExportDir(Path path) {
   }
 
   public ExportConfiguration setPemFile(Path path) {
-    this.pemFile = Optional.ofNullable(path);
+    //Check if path is a valid pem file with privateKey
+    if (PrivateKeyUtils.isValidPrivateKey(path)) {
+      this.pemFile = Optional.ofNullable(path);
+    }
     return this;
```

Comment:
Did you consider to throw a `BriefcaseException`? How do third parties know that their call to `ExportConfiguration.setPemFile(Path path)` didn't have any effect on it?